include/SendButtonIndicator.h

```diff
@@ -37,22 +37,20 @@ class FloatModel;
 namespace gui
 {
 
-class MixerLine;
+class MixerChannelView;
 class MixerView;
 
-
-class SendButtonIndicator : public QLabel 
+class SendButtonIndicator : public QLabel
 {
 public:
-	SendButtonIndicator( QWidget * _parent, MixerLine * _owner,
-						 MixerView * _mv);
+	SendButtonIndicator(QWidget * parent, MixerChannelView * owner, MixerView * mv);
 
-	void mousePressEvent( QMouseEvent * e ) override;
+	void mousePressEvent(QMouseEvent * e) override;
 	void updateLightStatus();
 
 private:
 
-	MixerLine * m_parent;
+	MixerChannelView * m_parent;
```

Comment:
Extra spaces between types and `*`

Reply:
Fixed with commit 92878ce4030. Using reference/pointer at type like in other places in the code.